libs/ngxtension/signal-slice/src/signal-slice.ts

```diff
@@ -157,16 +131,7 @@ export function signalSlice<
 	effects?: (
```

Comment:
nit: we can use `/** @deprecated */` JSDoc here to make the intention of deprecating `effects` clearer

Reply:
@nartc done